src/main.js

```diff
@@ -330,6 +330,32 @@ function createDefaultArgvConfigSync(argvConfigPath) {
 }
 
 function getArgvConfigPath() {
+	const oldArgvFilePath = getOldArgvConfigPath();
+	const newArgvFilePath = path.join(userDataPath, 'argv.json');
+
+	if (oldArgvFilePath !== newArgvFilePath) {
+		migrateArgvConfigPath(oldArgvFilePath, newArgvFilePath);
+	}
+
+	return newArgvFilePath;
+}
+
+/**
+ * Migrates the old argv path to the new path
+ * @param {string} oldFilePath
+ * @param {string} newFilePath
+ */
+function migrateArgvConfigPath(oldFilePath, newFilePath) {
+	if (!fs.existsSync(oldFilePath)) return;
+
+	if (fs.existsSync(newFilePath)) { // already exists
+		fs.unlinkSync(oldFilePath);
+	} else { // move the old file to a new location
+		fs.renameSync(oldFilePath, newFilePath);
```

Comment:
if `fs.renameSync` failed for some reason, should we display an error + exit cleanly instead of crashing?

Reply:
+1 might be permission issue